pkg/pgtune/parallel.go

```diff
@@ -72,8 +73,9 @@ func (r *ParallelRecommender) Recommend(key string) string {
 
 // ParallelSettingsGroup is the SettingsGroup to represent parallelism settings.
 type ParallelSettingsGroup struct {
-	pgVersion string
-	cpus      int
+	pgVersion    string
+	cpus         int
+	MaxBGWorkers int
```

Comment:
no need to be exported

Reply:
as discussed, I can't seem to write new tests without doing this